webapp/src/main/resources/application.sql.yml

```diff
@@ -67,3 +68,13 @@ sql:
 
     sqlFindByCreator: >-
       SELECT * FROM upload_student_group_batch WHERE creator=:creator
+
+  import:
+    createQuery: >-
+      INSERT INTO import (status, content, contentType, digest, batch) VALUES
+        (0, 5, 'admin group modification', :digest, :batch)
```

Comment:
I wasn't sure what to actually shove in there, the digest, or batch columns since this isn't a file-based import...  I feel like I'm breaking assumptions, but I don't have a better short-term alternative.